numpy/core/src/multiarray/descriptor.c

```diff
@@ -1309,18 +1309,9 @@ _convert_from_type(PyObject *obj) {
     if (PyType_IsSubtype(typ, &PyGenericArrType_Type)) {
         return PyArray_DescrFromTypeObject(obj);
     }
-#if !defined(NPY_PY3K)
```

Comment:
This avoided being cleaned up because of merge conflicts along the way.

Reply:
Guilty